pkg/auth/requests/authenticate.go

```diff
@@ -160,9 +176,69 @@ func (a *tokenAuthenticator) Authenticate(req *http.Request) (*AuthenticatorResp
 	authResp.Extras = getUserExtraInfo(token, u, attribs)
 	logrus.Debugf("Extras returned %v", authResp.Extras)
 
+	logrus.Debugf("Token %v, lastUsedAt: started update of `lastused`", token.ObjectMeta.Name)
+
+	// While the fix in the previous commit should have made this impossible maybe better to keep a guard.
+	if a.tokenWClient == nil {
+		logrus.Errorf("Token %v, lastUsedAt: skipping update, no wrangler client", token.ObjectMeta.Name)
+		return authResp, nil
+	}
+
+	now := time.Now().Truncate(lastUsedAtGranularity)
+	logrus.Debugf("Token %v, lastUsedAt: now is       %v", token.ObjectMeta.Name, now)
```

Comment:
Do we need this whitespace between "is" and "%v" ?

Reply:
Not really. This and the next log statement were initially near and I like vertically aligned output.
Will change.

Reply:
Addressed with commit 527b9883773520b17e5d7f4b5d2dc129b7f92b2c